Sources/MapboxNavigation/NavigationCameraStateTransition.swift

```diff
@@ -195,20 +195,20 @@ public class NavigationCameraStateTransition: CameraStateTransition {
             return
         }
         
-        let centerTranslationDistance = mapView.centerCoordinate.distance(to: location)
+        let centerTranslationDistance: CLLocationDistance = mapView.cameraState.center.distance(to: location)
         let metersPerSecondMaxCenterAnimation: Double = 1500.0
         let centerAnimationDuration: TimeInterval = max(min(centerTranslationDistance / metersPerSecondMaxCenterAnimation, 1.6), 0.6)
         let centerAnimationDelay: TimeInterval = 0.0
         
-        let zoomLevelDistance = CLLocationDistance(abs(mapView.zoom - zoom))
+        let zoomLevelDistance: CLLocationDistance = CLLocationDistance(abs(mapView.cameraState.zoom - zoom))
```

Comment:
shouldn't we rely on type inference here?

Reply:
I guess either one is fine. Added explicit type in this case just for the sake of match with other properties.